src/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -212,14 +228,35 @@ export class DashboardListing extends React.Component {
       return (
         <React.Fragment>
           <EuiCallOut
-            title="Listing limit exceeded"
+            title={this.props.intl.formatMessage('kbn.dashboard.listing.listingLimitExceeded.title', {
+              defaultMessage: 'Listing limit exceeded',
+            })}
             color="warning"
             iconType="help"
           >
             <p>
-              You have {this.state.totalDashboards} dashboards,
-              but your <strong>listingLimit</strong> setting prevents the table below from displaying more than {this.props.listingLimit}.
-              You can change this setting under <EuiLink href="#/management/kibana/settings">Advanced Settings</EuiLink>.
+              {this.props.intl.formatMessage('kbn.dashboard.listing.listingLimitExceeded.description', {
+                defaultMessage: 'You have {totalDashboards} dashboards, \
```

Comment:
Please do not use `\`, because browsers behave different with multiline javascript strings by using `\`.
Moreover in jsx it's necessary to use `\` for multiline string in template. This should work:
```
defaultMessage: 'You have {totalDashboards} dashboards, but your {listingLimitText} setting prevents the table below from displaying more than {listingLimitValue}. You can change this setting under {advancedSettingsLink}.'
```